flashinfer/fused_moe/cute_dsl/blackwell_sm12x/_moe_utils/moe_epilogue.py

```
@@ -0,0 +1,869 @@
+    QUANT_AT = None
+    DRAINS_SC_IN_WG = False
+    _PERIOD_BYTES = {
+        cute.nvgpu.warpgroup.SmemLayoutAtomKind.K_SW128: 128,
+        cute.nvgpu.warpgroup.SmemLayoutAtomKind.K_SW64: 64,
+        cute.nvgpu.warpgroup.SmemLayoutAtomKind.K_SW32: 32,
+        cute.nvgpu.warpgroup.SmemLayoutAtomKind.K_INTER: STG_BYTES,
+        cute.nvgpu.warpgroup.SmemLayoutAtomKind.MN_SW128: 128,
+        cute.nvgpu.warpgroup.SmemLayoutAtomKind.MN_SW64: 64,
+        cute.nvgpu.warpgroup.SmemLayoutAtomKind.MN_SW32: 32,
+        cute.nvgpu.warpgroup.SmemLayoutAtomKind.MN_INTER: STG_BYTES,
+    }
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)